diff --git a/modules/libcom/src/error/errlog.c b/modules/libcom/src/error/errlog.c
index 75c6e63c1..ead0f5d29 100644
--- a/modules/libcom/src/error/errlog.c
+++ b/modules/libcom/src/error/errlog.c
@@ -41,8 +41,9 @@
 #include "osiUnistd.h"
 
 
-#define MIN_BUFFER_SIZE 1280
+#define MIN_BUFFER_SIZE 2560
 #define MIN_MESSAGE_SIZE 256
+#define DEFAULT_MAX_MESSAGE_SIZE 512
 #define MAX_MESSAGE_SIZE 0x00ffffff
 
 /* errlog buffers contain null terminated strings, each prefixed
@@ -608,7 +609,7 @@ int errlogInit2(int bufsize, int maxMsgSize)
 
 int errlogInit(int bufsize)
 {
-    return errlogInit2(bufsize, MIN_MESSAGE_SIZE);
+    return errlogInit2(bufsize, DEFAULT_MAX_MESSAGE_SIZE);
 }
 
 void errlogFlush(void)
